cmd/amp/commands.go

```diff
@@ -53,7 +57,7 @@ func newRootCommand(c cli.Interface) *cobra.Command {
 	cli.SetupRootCommand(cmd)
 
 	cmd.Flags().BoolVarP(&opts.version, "version", "v", false, "Print version information and quit")
-	cmd.PersistentFlags().StringVarP(&opts.addr, "server", "s", "", "Specify server (host:port)")
+	cmd.PersistentFlags().StringVarP(&opts.addr, "server", "s", "cloud.atomiq.io:50101", "Specify server (host:port)")
```

Comment:
We only have a one single location for updating the default server to connect to. There should be no default for the flag here.